### ISSUE TITLE
Remove outline from elements removed from tab flow

### ISSUE THREAD

**Issue by necolas** [suitcss/base#2]
``` css
/**
 * Suppress the focus outline on links that cannot be accessed via keyboard.
 * Prevents the dotted outline from persisting around some block links that
 * open in a new window.
 */

[tabindex="-1"] {
  outline: none;
}
```
